contrib/parity_games/solve_parity_game.py

```diff
@@ -1,110 +1,107 @@
-"""
-McNaughton's algorithm for solving parity games
+"""McNaughton's algorithm for solving parity games.
+
 
 Reference
 =========
+
 Robert McNaughton (1993). Infinite games played on finite graphs.
 Annals of Pure and Applied Logic 65(2): 149--184.
-doi:10.1016/0168-0072(93)90036-D
+<doi:10.1016/0168-0072(93)90036-D>
```

Comment:
```suggestion
<https://doi.org/10.1016/0168-0072(93)90036-D>
```
or `[doi:10.1016/0168-0072(93)90036-D](https://doi.org/10.1016/0168-0072(93)90036-D)`. Anyway, `doi` is not a scheme recognized by Chrome nor Firefox. When I try `doi:10.1016/0168-0072(93)90036-D` in both browsers, they search for it, and the top result from DuckDuckGo has the title "Carcass feeding of captive cheetahs".